<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>insert</class>
 <widget class="QDialog" name="insert">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>345</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>18</pointsize>
       <weight>75</weight>
       <italic>false</italic>
       <bold>true</bold>
       <strikeout>false</strikeout>
      </font>
     </property>
     <property name="text">
      <string>Insert a person</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_nameperson">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_nameperson">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_genderperson">
     <property name="text">
      <string>Gender</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_genderperson"/>
   </item>
   <item>
    <widget class="QLabel" name="label_yearbornperson">
     <property name="text">
      <string>Year born</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_yearbornperson"/>
   </item>
   <item>
    <widget class="QLabel" name="label_yearofdeath">
     <property name="text">
      <string>Year of death</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="lineEdit_yeardeathperson"/>
   </item>
   <item>
    <widget class="QPushButton" name="pushButton_insert">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
